resources/public/clojure-reload-dbg.html

```diff
@@ -0,0 +1,37 @@
+
+<!DOCTYPE html>
+<html>
+  <head lang="en">
+    <meta charset="UTF-8">
+    <title>KLIPSE: a simple and elegant online cljs compiler and evaluator</title>
+    <link rel='shortcut icon' type='image/x-icon' href='img/klipse.png' />
+    <link rel="stylesheet" type="text/css" href="css/codemirror.css">
+  </head>
+  <body>
+    <div class="clojure">
+      (def x 1)
+      x
+    </div>
+    <div class="clojure">
+      (def y (inc x))
+      y
+    </div>
+    <div class="clojure">
+      (def z (dec (dec y)))
+      z
+    </div>
+    <script>
+      window.klipse_settings = {
+        eval_idle_msec: 1500,
+        re_evaluate_all_snippets_on_change: true
+      };
+      if(window.location.search.match(/transpile=1/)) {
+      window.klipse_settings.selector_js = '.clojure';
+      } else {
+      window.klipse_settings.selector = '.clojure';
```

Comment:
Could you create a test page that mixes snippets from different languages and make sure it work as expected?